LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2577,6 +2579,52 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
                         c.Add(new Control("Chassis Fan #3", 6));
                         break;
 
+                    case Model.Z790_Nova_WiFi:
+                        if (index != 0)
+                        {
+                            // second SIO
+                            v.Add(new Voltage("0.82V Chipset", 4));
+                            v.Add(new Voltage("1.05V CPU", 0));
+                            v.Add(new Voltage("VDD_CPU", 1));
+                            v.Add(new Voltage("1.05V Chispet", 12, 5, 100));
+
+                            f.Add(new Fan("VRM", 0));
+                            f.Add(new Fan("MOS", 1));
+
+                            c.Add(new Control("VRM", 0));
+                            c.Add(new Control("MOS", 1));
+                        }
+                        else
+                        {
+                            v.Add(new Voltage("+12V", 4, 110, 10));
+                            v.Add(new Voltage("+5V", 1, 20, 10));
+                            v.Add(new Voltage("+3.3V", 3, 34, 34));
+                            v.Add(new Voltage("VCCIN_AUX", 5, 1, 1));
```

Comment:
I’m noticing that the voltages entry for the first SuperIO seems to be incomplete.

I'm convinced that the voltage entry for the first SuperIO chip would be similar to #1387. I'm curious if that ends up being the case. Mind checking that out?

Also, what are you using to validate voltage sensors? If HWiNFO64 is included, I'm curious what the entire voltage indexes on the second SuperIO from index 0-15 contains since that hasn't been shown in the PR yet.

Reply:
I just forgot to add CPU System Agent and Vcore, thanks for your notice.

I'm using the HW monitor values in BIOS setup to validate those readings. I think not all the indexes on 2nd SIO are using for this mobo.

Reply:
HW Monitor in the UEFI is definitely good for making sure the ones that ASRock actually present are correct

However I tend to also use HWiNFO64 to also have a cross check for sensors that ASRock port internally, but don’t show in the UEFI

For example. It looks like before your data entry the CMOS Battery, +3V Standby were accurate, and VTT needed to be 1:1

Voltage index 6 on the first SIO might be the IMC, and Index 14 might be +5V Standby (check my PR too, you’ll likely need to change to 2.35,1)